guide/C/ch_basics.xml

```diff
@@ -483,7 +483,7 @@ Translators:
         in the <guilabel>Welcome to &appname;!</guilabel> menu, or when you select <menuchoice>
           <shortcut>
           <keycombo>
-            <keycap>Strg</keycap><keycap>N</keycap>
+            <keycap>Ctrl</keycap><keycap>N</keycap>
```

Comment:
Right, this happened, when we backported improvements from the German translation.

Currently only translators are working on the docs.

Reply:
I think shortcuts and acceleration keys are not necessary in this guide.
Write them in the help manual instead.